### PR TITLE
Create NOTICE.txt for base microservice

### PR DESCRIPTION
**Pull request by KieranFrance**

### CODE REVIEW

**Comment by KieranFrance** [comment]
I have created a Notice.txt file to contain the copyright and removed the line from all the source files.
I decided to keep the license information in the classes and just removed the copyright year.

The intention is to make updating the copyright years simpler

File: NOTICE.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+                        Base Microservice Example
+                        =========================
+
+Please visit the IDRSolutions implementations of the base microservice for examples of this project in use:
+  * https://github.com/idrsolutions/buildvu-microservice-example
+  * https://github.com/idrsolutions/formvu-microservice-example
+  * https://github.com/idrsolutions/jpedal-microservice-example
+
+Copyright 2018 - 2023 IDRsolutions
```
Comment (leonatherton) [inline NOTICE.txt:9]: I'm thinking in this one we should have a mention regarding the DefaultFileServlet class which is derived from OmniFaces.

Comment (KieranFrance) [reply NOTICE.txt:9]: I have left the OmniFaces copyright in that class.
Shall we also include it here as follows, I took inspiration from other projects.

Copyright 2018 - 2023 IDRsolutions
This project includes work covered by the following copyright:
DefaultFileServlet.java - Copyright 2021 OmniFaces

File: NOTICE.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Base Microservice Example
+
+Copyright 2018 - 2023 IDRsolutions
+This project includes work covered by the following copyright:
+DefaultFileServlet.java - Copyright 2021 OmniFaces
```
Comment (leonatherton) [inline NOTICE.txt:5]:
```suggestion
The DefaultFileServlet class is a derivation of FileServlet by OmniFaces.
Copyright 2021 OmniFaces
```

**Review by leonatherton — APPROVED** [review]

**Review by daniel-warren — APPROVED** [review]

**Review by Amy1612 — APPROVED** [review]

**Pull request merged**